src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -7147,6 +7156,19 @@ void wm_vuldet_init(wm_vuldet_t * vuldet) {
         // Initial sleep
         sleep(time_sleep);
     }
+
+    /* Get node name of this manager in cluster */
+    const char *(xml_node[]) = {"ossec_config", "cluster", "node_name", NULL};
+
+    OS_XML xml;
+
+    if (OS_ReadXML(DEFAULTCPATH, &xml) < 0){
+        mterror(WM_VULNDETECTOR_LOGTAG, XML_ERROR, DEFAULTCPATH, xml.err, xml.err_line);
+    }
+
+    node_name = OS_GetOneContentforElement(&xml, xml_node);
```

Comment:
I think this should be part of **wm_vuldet_t** struct

Reply:
Fixed.